### DISCUSSION TITLE
can Tree Node be an svg icon ?

### DISCUSSION THREAD

**Discussion by Jasminou** [airbnb/visx#1391] [Q&A]
in your Tree example is it possible to have an icon in node ?

**Comment by williaster** [comment]
converting this to a discussion as it's a question not an issue.

and yes, you can render anything as a node, `svg` nodes are valid children of other `svg` nodes.

**Reply by Jasminou** [reply]
thanks for the quick answer, is it possible to have example with icon/image as node ?

**Reply by williaster** [reply]
I don't have a ton of time at the moment to add an example, but maybe someone else can or you can check out this related discussion https://github.com/airbnb/visx/discussions/1307

**Reply by Jasminou** [reply]
Thank you that helps. Another question for you if you don't mind. is it Possible to change the Tree drawing orientation. instead from left to right to have it from right to left ?

**Reply by williaster** [reply]
yes you should just be able to reverse the `x` value
